### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by Elisha-Tetteyfio**
In this milestone, myself and @henrycode460  refactored the methods to include methods to store data locally.
- Method to save data for `people`
- Method to save data for `books`
- Method to save data for `rentals`

### CODE REVIEW

File: create_person.rb [hunk 1]
```diff
@@ -17,8 +17,15 @@ def create_person
...
-      puts " #{new_student.name} added successfully"
+      student_obj = {
+        Name: new_student.name,
+        Age: new_student.age,
+        Classroom: new_student.classroom,
+        ID: new_student.id,
+        Person: new_student.class,
+        Rentals: []
+      }
+      @people << student_obj
```
Comment (Nemwel-Boniface) [inline create_person.rb:28]: - [ ] Great job saving and retrieving data. The application saves and works mostly as expected. The only issue comes when I already have some data in the "database" (.json) files on app start and I add some more data, the original data gets overwritten and now only the new data is present in the .json files. Kindly look into this.

[Screencast from 13-09-2022 10:10:37 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/189990208-773ddf05-b7b1-498e-bea5-1a796f068e57.webm)

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Elisha-Tetteyfio and @henrycode460       :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your project so far :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Elisha-Tetteyfio** [comment]
Thank you @Nemwel-Boniface , we will review that.

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Elisha-Tetteyfio and @henrycode460   🔥 

Good job on implementing the required changes. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**